ftplugin/gnuplot.vim

```diff
@@ -7,3 +7,14 @@ setlocal commentstring=#\ %s
 setlocal completefunc=syntaxcomplete#Complete
 
 compiler gnuplot
+
+if executable('gnuplot')
+  if !has('gui_running') && executable('less')
+    command! -buffer -nargs=1 RunHelp silent exe '!gnuplot -e "help <args>" 2>/dev/null' | redraw!
+  elseif has('terminal')
+    command! -buffer -nargs=1 RunHelp silent exe ':term gnuplot -e "help <args>"'
+  else
+    command! -buffer -nargs=1 RunHelp echo system('gnuplot -e "help <args>" 2>/dev/null')
+  endif
+  setlocal keywordprg=:RunHelp
```

Comment:
Where does this `RunHelp` command comes from ?

Reply:
It is a custom command set up in the lines before. This is necessary, because `gnuplot -e help keyword` is not properly quoted, it must be `gnuplot -e "help keyword"`. The command achieves this, whereas `&l:keywordprg = 'gnuplot -e help' will not quote the keyword properly.

I am just collecting compilers. Gnuplot is a program that, while not using at the moment, always pays dividends when there is something more involved to plot.

While one could fork, I am not more active a user and a newcomer. This repository is the first address to turn to and therefore still the best place to host such improvements.

Reply:
Oh right, I thought that the command was launched in the if branches directly ! Good for me then :)